ui/js/graphs.js

```diff
@@ -669,14 +669,26 @@ perf.controller('GraphsCtrl', [
                 ThRepositoryModel.get_list().then(function(response) {
 
                     $scope.projects = response.data;
-                    $scope.addTestData = function() {
+                    $scope.addRelatedPlatform = function(seriesSignature) {
+                        $scope.addTestData("addRelatedPlatform", seriesSignature);
```

Comment:
I don't think it's a big deal, but it might be nice to make the parameter an enumeration using the technique described here: http://stackoverflow.com/a/2383215

I'll leave this up to you. :)